### DISCUSSION TITLE
Hide bans, when they happened before first game with them

### DISCUSSION THREAD

**Discussion by Flashbert** [akiver/cs-demo-manager#760] [Ideas]
For now it shows all bans, even they happened before i played with them, so i see very old VAC bans (probably from other games)

Maybe make an option to only show bans, if they happened after your (first) game with/against them.

Bonus idea:
If you are on a profile, or on the VAC ban page, create a rightclick option to add to Ban-Ignore-List

**Comment by akiver** [comment]
Available as of v3.3.1
